core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -763,9 +763,32 @@ class JavaSparkContext(val sc: SparkContext) extends Closeable {
   /**
    * Cancel active jobs for the specified group. See
    * `org.apache.spark.api.java.JavaSparkContext.setJobGroup` for more information.
+   *
+   * @param groupId the group ID to cancel
+   * @param reason reason for cancellation
+   *
+   * @since 4.0.0
+   */
+  def cancelJobGroup(groupId: String, reason: String): Unit = sc.cancelJobGroup(groupId, reason)
```

Comment:
given that the recently added `cancelJobGroupAndFutureJobs` is not here, this API doesn't have to be here as well.